File: javascript/control_flow/continue_instruction/continue_instruction.js
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

let i = 0;

while (i < 10) {
  console.log(`${i++}...`);

  if (i > 5)
    continue;

  console.log("*********************");
}

File: python/control_flow/continue_instruction/continue_instruction.py
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
# SPDX-License-Identifier: GPL-3.0-or-later

number = 0

while number < 10:
    print(f'{number}...')
    number += 1

    if number > 5:
        continue

    print("*********************")
